probe-rs/targets/nRF91_Series.yaml

```diff
@@ -144,7 +174,7 @@ flash_algorithms:
     address_range:
       start: 0xff8000
       end: 0xff9000
-    page_size: 0x1000
+    page_size: 4
```

Comment:
Why do you set the page size to 4 here? Doesn't 0x1000 make more sense, as an erase will erase the whole UICR?

Reply:
page_size is the minimum amount it can be written, not erased. Erase size is the sector size.

Reply:
> Larger sizes were causing problems because nrf91 actually can't erase the uicr at all. The algo's sector_erase is a noop.

UICR can be erased, using the erase all functionality.

Reply:
yes, but there's no way to erase *only* UICR. The algo implements it as a noop because the only alternative it has is nuking everything, and probe-rs has no idea it's a noop.